.github/ISSUE_TEMPLATE/10-issue_report.yaml

```diff
@@ -0,0 +1,136 @@
+name: Report a Bug
+title: "<title>"
+description: |
+  You found a bug in Bareos causing the application to crash or
+  throw an exception, a component is buggy, or something looks wrong.
+labels: needs triage
+body:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Bareos and taking the time to fill out this bug report!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on mailing list:
+
+          - https://doc.bareos.org/
+          - https://groups.google.com/u/1/g/bareos-users
+  - type: markdown
+    attributes:
+      value: |
+        Before filling the form fields, please consider the following:
+        - Ensure that you have searched the [existing issues](https://github.com/bareos/bareos/issues)
+        - Read the [troubleshooting guide](https://docs.bareos.org/Appendix/Troubleshooting.html)
+        - Read the [guide to filing a bug](https://docs.bareos.org/Appendix/Troubleshooting.html#bug-reports)
+
+        **What is a bug?** A bug describe a unexpected trouble with the software which
+        prove a divergence with the documentation, and/or a crash (trace) that are
+        somewhat reproducible.
+
+        We always recommend to start discussing your issue on the mailing list.
+  - type: dropdown
+    id: bareos_version
+    attributes:
+      label: Bareos version
+      multiple: false
+      description: What version of Bareos are you seeing the problem on?
+      options:
+        - 23x
+        - 24x
+        - Others
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Please tell us exactly how to reproduce the problem you are running into.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+
+        Note: Please try to ensure the bug is reproducible on another system!
```

Comment:
This placeholder is too long.  At least i cannot see the `Note:` sentence!

Reply:
I will shorten it